src/primevue/select/select.ts

```diff
@@ -0,0 +1,48 @@
+import { SelectPassThroughOptions } from "primevue/select";
+import { tw } from "@/lib/tags.ts";
+import "./select.css";
+
+export const base = tw`ris-body2-regular h-full min-h-48 w-full border-2 border-blue-800 bg-white px-16 py-4 outline-4 -outline-offset-4 outline-blue-800 placeholder:text-gray-600 hover:outline focus:outline disabled:border-blue-500 disabled:bg-white disabled:text-blue-500 disabled:outline-none [&+label]:my-4`;
```

Comment:
Todo: please remove the `h-full` and `w-full` classes, the sizing of inline components like this should be decided by the parent using the component. Making this always take the full available space restricts how the component can be used. Instead:

- The component should be displayed as inline with automatic width
- Make it full width based on the [`fluid` prop](https://primevue.org/select/#api.select.props.fluid). This is a prop used by many components in PrimeVue. You can refer to our input text preset for an example. Also see https://primevue.org/fluid/